pkg/enqueue-bundle/DependencyInjection/EnqueueExtension.php

```diff
@@ -171,6 +171,14 @@ private function registerJobQueueDoctrineEntityMapping(ContainerBuilder $contain
             return;
         }
 
+        foreach ($container->getExtensionConfig('enqueue') as $modules) {
+            foreach ($modules as $config) {
+                if (isset($config['job']) && false === $config['job']['default_mapping']) {
```

Comment:
That does not look accurate. For example one config defines `false` and the next defines `true`. The merged value might be `true` but this logic would not load default entites.

There must be a method that allows getting merged version of the config. But I am not sure. Could you please look for it?

Reply:
Sorry for taking so long.
I was not able to find a solution for this yet, but this part of code is happening before the merging of configuration, no?

Reply:
You should merge it manually and use the result in condition.

Reply:
Symfony provides tools for that.

Reply:
I believe

Reply:
https://symfony.com/doc/current/bundles/configuration.html#processing-the-configs-array